### PR TITLE
[forge] Add account creation continuous forge test and having loadtest params per test

### PR DESCRIPTION
**Pull request by igor-aptos**
- Moved duration out of load test params, and into Forge.
- load tests is now specified together with SuccessCriteria, and not via CLI as duration is.

This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3609)

### CODE REVIEW

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -442,6 +437,25 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
...
+            .with_network_tests(&[&PerformanceBenchmarkWithFN])
+            .with_initial_validator_count(NonZeroUsize::new(5).unwrap())
+            .with_initial_fullnode_count(3)
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 1200.into();
+            }))
+            .with_emit_job(
+                EmitJobRequest::default()
+                    .mode(EmitJobMode::MaxLoad {
+                        mempool_backlog: 50000,
```
Comment (sitalkedia) [inline testsuite/forge-cli/src/main.rs:450]: 50K is probably too much - I have seen the emitter destabilizing with so high load. Let's make it between 30 to 40k?

**Review by sitalkedia — APPROVED** [review]
LGTM

**Review by rustielin — APPROVED** [review]

File: testsuite/forge-cli/src/main.rs [hunk 2]
```diff
@@ -442,6 +437,25 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 10000,
                 Some(Duration::from_secs(240)),
             )),
+        "account_creation_state_sync" => config
```
Comment (sitalkedia) [inline testsuite/forge-cli/src/main.rs:440]: @igor-aptos - Do you want to create a test for mix transaction set up as well?

Comment (igor-aptos) [reply testsuite/forge-cli/src/main.rs:440]: Yes,  separate PR.

How do I make this one run in continuous?

**Pull request merged**